### PR TITLE
Add inside_ip to node and populate it in topo.Show

### PR DESCRIPTION
**Pull request by will2dye4**
This is needed for KNE clients to be able to configure the node's cluster-internal IP on the management interface for vendors that do not automatically configure the management interface with the node's IP.

### CODE REVIEW

File: proto/topo.proto [hunk 1]
```diff
@@ -87,6 +87,8 @@ message Node {
   // Any host constraints required by the node. These may not be required by a
   // vendor, but it can be used to provide requirements for the node.
   repeated HostConstraint host_constraints = 13;
+  // Cluster IP for the node's pod. Assigned by KNE.
```
Comment (marcushines) [inline proto/topo.proto:90]: I would suggest: pod_ip

as that is more accurate

IP assigned by k8s for the pod this ip comes from the worker node's IP pool

Comment (will2dye4) [reply proto/topo.proto:90]: I chose `inside_ip` and phrased the comment to mirror the existing field on the `Service` proto message:

```
  string inside_ip = 4;   // Cluster IP for the service.
```

I think it's important to convey, either in the name or in comments or both, that this is an *internal* IP address, i.e., this IP cannot be used to reach the node outside of the cluster.

Comment (marcushines) [reply proto/topo.proto:90]: while that "sounds good" inside ip and pod ip are actually different constructs

one ip is node worker specific another a common ip in the whole cluster that is used by the loadbalancer they are in fact different ip's and cluster inside ip means something 

as well as pod ip

Comment (will2dye4) [reply proto/topo.proto]: That makes sense. I changed the name accordingly.

**Review by marcushines — APPROVED** [review]

**Pull request merged**